internal/pkg/describe/status.go

```diff
@@ -14,14 +14,13 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/aws/aas"
 	"github.com/aws/copilot-cli/internal/pkg/aws/cloudwatch"
 	"github.com/aws/copilot-cli/internal/pkg/aws/ecs"
-	rg "github.com/aws/copilot-cli/internal/pkg/aws/resourcegroups"
 	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
+	copiEcs "github.com/aws/copilot-cli/internal/pkg/ecs"
```

Comment:
nit: I'd instead rename the `aws/ecs` to `awsECS`. Reading "copiEcs" is kinda odd :P